test/unit/BLSSignatureCheckerUnit.t.sol

```diff
@@ -9,28 +9,53 @@ contract BLSSignatureCheckerUnitTests is BLSMockAVSDeployer {
 
     BLSSignatureChecker blsSignatureChecker;
 
+    event StaleStakesForbiddenUpdate(bool value);   
+
     function setUp() virtual public {
         _setUpBLSMockAVSDeployer();
 
         blsSignatureChecker = new BLSSignatureChecker(registryCoordinator);
     }
 
+    function test_setStaleStakesForbidden_revert_notRegCoordOwner() public {
+        cheats.expectRevert("BLSSignatureChecker.onlyCoordinatorOwner: caller is not the owner of the registryCoordinator");
+        blsSignatureChecker.setStaleStakesForbidden(true);
+    }
+
+    function test_setStaleStakesForbidden() public {
+        testFuzz_setStaleStakesForbidden(false);
+        testFuzz_setStaleStakesForbidden(true);
+    }
+
+    function testFuzz_setStaleStakesForbidden(bool newState) public {
```

Comment:
Is there a point to making this a fuzz test? There are only 2 values that can be provided, and the test above tries them both out directly